src/api/get-installations.js

```diff
@@ -8,6 +8,24 @@ import ALIASES from '../aliases';
 // Installation info cache
 var installationsCache = null;
 
+// Original code page value
+var origCodePage = null;
```

Comment:
Can we do by that way ?
[simplified scheme](https://gist.github.com/georgiy-abbasov/98db1990a7e6ea48bd6d)